packages/lib/CalendarService.ts

```diff
@@ -190,30 +190,56 @@ export default abstract class BaseCalendarService implements Calendar {
 
       const eventsToUpdate = events.filter((e) => e.uid === uid);
       return Promise.all(
-        eventsToUpdate.map((e) => {
+        eventsToUpdate.map((eventItem) => {
+          event = eventItem;
           return updateCalendarObject({
             calendarObject: {
-              url: e.url,
-              data: iCalString,
-              etag: e?.etag,
+              url: event.url,
+              data: iCalString?.replace(/METHOD:[^\r\n]+\r\n/g, ""),
```

Comment:
Adds the adjustment to comply with iCal2.0 standard which various service providers like nextcloud and fastmail expect